docs/PowerAuth-2019.05.md

```diff
@@ -0,0 +1,72 @@
+# PowerAuth 2019.05
+
+## Migration guides
+
+For updating to 2019.05, please follow these migration guides:
+
+- [PowerAuth Server - Migration from version 0.21.0 to version 0.22.0](https://github.com/wultra/powerauth-server/blob/develop/docs/PowerAuth-Server-0.22.0.md)
+- [PowerAuth Push Server - Migration from version 0.21.0 to version 0.22.0](https://github.com/wultra/powerauth-push-server/blob/develop/docs/PowerAuth-Push-Server-0.22.0.md)
+- [PowerAuth Web Flow - Migration from version 0.20.0 to version 0.21.0](https://github.com/wultra/powerauth-webflow/blob/develop/docs/Web-Flow-0.22.0.md)
+- [PowerAuth Mobile SDK - Migration from version 1.0.0 to version 1.1.0](https://github.com/wultra/powerauth-mobile-sdk/blob/develop/docs/Migration-from-1.0-to-1.1.md)
+
+## Components for version 2019.05
+
+### Back-End Applications
+
+| Component | Application Name | Version | Description |
+|---|---|---|---|
+| PowerAuth Server | `powerauth-java-server.war` | 0.22.0 | Core back-end component for PowerAuth stack. |
+| PowerAuth Admin | `powerauth-admin.war` | 0.22.0 | Administration console for PowerAuth Server. |
+| PowerAuth Push Server | `powerauth-push-server.war` | 0.22.0 | Simple to deploy push server for APNS and FCM. |
+| PowerAuth Web Flow | `powerauth-webflow.war` | 0.22.0 | Central web authentication page. |
+| PowerAuth Next Step | `powerauth-next-step.war` | 0.22.0 | Simple authorization server used for PowerAuth Web Flow component. |
+| PowerAuth Data Adapter | `powerauth-data-adapter.war` | 0.22.0 | Customization component for PowerAuth Web Flow. |
+
+### Utilities
+
+| Component | Application Name | Version | Description |
+|---|---|---|---|
+| PowerAuth Command Line Tool | `powerauth-java-cmd.jar` | 0.22.0 | Command line tool for integration testing. |
+
+### Mobile Libraries
+
+| Platform | Package Name | Version | Description |
+|---|---|---|---|
+| iOS | `PowerAuth2` | 0.20.1 / 1.1.0 | A client library for iOS. |
+| watchOS | `PowerAuth2ForWatch` | 0.20.1 / 1.1.0 | A limited library for watchOS. |
+| iOS App Extensions | `PowerAuth2ForExtensions` | 0.20.1 / 1.1.0 | A limited library for iOS App Extensions. |
+| Android | `io.getlime.security.powerauth:powerauth-android-sdk` | 0.20.1 / 1.1.0 | A client library for Android. |
+
+_Note: Libraries with version `0.20.x` support PowerAuth protocol version `2`. Libraries with versions `1.0.x` and `1.1.x` support PowerAuth protocol version `3`._ 
```

Comment:
We don't need to mention `1.0.x` in this release, just `1.1.0`

Reply:
Ok, fixed.